client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -134,7 +135,10 @@ export default function getThankYouPageUrl( {
 	// (and WP.com's) paid blocks Upgrade Nudge.
 	if ( redirectTo ) {
 		const { protocol, hostname, port, pathname, searchParams } = getUrlParts( redirectTo );
-
+		if ( isCalypsoLive() ) {
+			debug( 'is calypso live, so returning that', redirectTo );
+			return redirectTo;
+		}
```

Comment:
I think we don't this anymore after fixing the URL. cc @escapemanuele .

Reply:
💯  

If you do need to modify this function, please also modify the tests to prove what you're trying to prevent and protect against future regressions (see the comments in this file). There may also be a better way to do it. All the code that creates redirect URLs already takes into account different host names and making an exception for calypso.live feels like it might be a hack to fix something more subtle.

Reply:
We do not need `isCalypsoLive` anymore luckily, removed 👍